app/src/main/kotlin/com/bitwarden/authenticator/ui/authenticator/feature/manualcodeentry/ManualCodeEntryViewModel.kt

```diff
@@ -133,6 +177,22 @@ class ManualCodeEntryViewModel @Inject constructor(
     }
 }
 
+private fun deriveButtonState(
+    sharedCodesState: SharedVerificationCodesState,
+    defaultSaveOption: DefaultSaveOption,
+): ManualCodeEntryState.ButtonState {
+    // If syncing with Bitwarden is not enabled, who local save only:
```

Comment:
```suggestion
    // If syncing with Bitwarden is not enabled, show local save only:
```

Reply:
Thank you! 397a172f0c8a150307a3f78e9e760d7f746b2011